src/generated/xproto.rs

```diff
@@ -4753,75 +5661,97 @@ impl TryParse for ClientMessageData {
     }
 }
 impl From<[u8; 20]> for ClientMessageData {
-    fn from(value: [u8; 20]) -> Self {
+    fn from(data8: [u8; 20]) -> Self {
+        let value = [
+            data8[0],
+            data8[1],
+            data8[2],
+            data8[3],
+            data8[4],
+            data8[5],
+            data8[6],
+            data8[7],
+            data8[8],
+            data8[9],
+            data8[10],
+            data8[11],
+            data8[12],
+            data8[13],
+            data8[14],
+            data8[15],
+            data8[16],
+            data8[17],
+            data8[18],
+            data8[19],
+        ];
         Self(value)
```

Comment:
If this is easy to fix: I like the old version of this piece of code better.
If this is not easy to fix: Feel free to keep it like this.

Reply:
Fixed